### PR TITLE
python3: xml writing seems to require Bytes, not String

### PR DESCRIPTION
**Pull request by keflavich**
All in the title.  I haven't tested on py2, but it definitely didn't work on py3.

Is it OK to have astropy as a requirement?  It simplifies the import.

### CODE REVIEW

**Comment by saimn** [comment]
I have a similar issue while trying to save a jpg file with aplpy: https://gist.github.com/saimn/c4d5865e590be2ab348cbc31f30d3005 (note the line with `StringIO()` which is modified by this PR).
I can try to look more in detail, but probably not in the next weeks.

File: pyavm/avm.py [hunk 1]
```diff
@@ -27,10 +27,7 @@
     basestring = unicode = str
 
 import warnings
-try:
-    from StringIO import StringIO
-except ImportError:
-    from io import BytesIO as StringIO
+from astropy.extern.six import BytesIO
```
Comment (astrofrog) [inline pyavm/avm.py]: What about using ``from io import BytesIO``?

**Comment by astrofrog** [comment]
@keflavich - could you rebase this and add a regression test? I'd rather not add a required astropy dependency if possible, so can you import ``BytesIO`` from ``io`` as mentioned above?

**Comment by astrofrog** [comment]
I finished this up in https://github.com/astrofrog/pyavm/pull/33

**Pull request closed without merging**